### PR TITLE
images: Install redis on CentOS/RHEL images

### PR DESCRIPTION
**Pull request by martinpitt**
Similar to commit a5fefffa9dc, to test integration with Grafana.

 * [x] image-refresh rhel-8-4
 * [x] image-refresh rhel-8-5
 * [x] image-refresh rhel-9-0
 * [x] image-refresh centos-8-stream

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-1995-20210506-074343/

**Comment by cockpituous** [comment]
image-refresh rhel-8-4 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-4-20210506-080505

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-1995-20210506-080606/

**Comment by cockpituous** [comment]
image-refresh rhel-8-5 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-5-20210506-082626

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-03.
Log: https://logs.cockpit-project.org/logs/image-refresh-1995-20210506-084747/

**Comment by cockpituous** [comment]
image-refresh rhel-9-0 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-9-0-20210506-090000

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-8.
Log: https://logs.cockpit-project.org/logs/image-refresh-1995-20210506-090101/

**Comment by cockpituous** [comment]
image-refresh centos-8-stream done: https://github.com/cockpit-project/bots/commits/image-refresh-centos-8-stream-20210506-093131

**Comment by martinpitt** [comment]
[single failure on rhel-9-0](https://logs.cockpit-project.org/logs/pull-1995-20210506-100323-80be9025-rhel-9-0-cockpit-project-cockpit/log.html#230), which looks familiar. Retrying once to compare.

**Comment by martinpitt** [comment]
Meh, nothing is ever easy.. NTP is broken on RHEL 9.0 now, and then the last refresh in #1989 just happened yesterday *cry*

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -156,6 +156,7 @@ ltrace \
 nmap-ncat \
 python3-tracer \
 qemu-kvm \
+redis \
```
Comment (marusak) [inline images/scripts/rhel.setup:159]: No need to do the same?

https://github.com/cockpit-project/bots/commit/16ec1980bdc4b333ec2de793a85eedd1dd7511e3#diff-b0928b5227c29cd0b5ef960d71d5bb357c1f1831de3a3a5148b74910386ad86dR191

Comment (martinpitt) [reply images/scripts/rhel.setup:159]: No, services don't auto-start by default in Fedora/RHEL.

Comment (martinpitt) [reply images/scripts/rhel.setup:159]: ... in fact, that cost me half an hour of my life as when it's not running, the pmproxy API requests hang forever :grin:

**Review by KKoukiou — APPROVED** [review]

File: images/scripts/rhel.setup (hunk 1 above)
Comment (KKoukiou) [reply images/scripts/rhel.setup:159]: interesting - I had no idea

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**